xdsl/pattern_rewriter.py

```diff
@@ -39,8 +39,7 @@ class PatternRewriterListener(BuilderListener):
     """Callbacks that are called when an operation is removed."""
 
     operation_modification_handler: list[Callable[[Operation], None]] = field(
-        default_factory=list,
-        kw_only=True,
+        default_factory=list, kw_only=True
     )
```

Comment:
This change looks weird :thinking: 
Is this black acting up?

Reply:
It was me, because it wasn't consistent with the field right on top of it.
To be fair, I'd rather add this in this PR than adding a completely separate PR just for this typo, as you prefer.